Docs: Add instructions for renaming and managing codespaces (closes #229) (#250)
r-devel/r-dev-env: docs/container_setup/github_codespace/creating_codespace.md

3. You will be taken to a VSCode editor within your browser.
![codespace](../../assets/rdev3.png)

> 4. Rename your codespace for easier identification:

Reviewer: ```suggestion
4. Rename your codespace for easier identification.
```

> By default, your codespace will have a randomly generated name. Renaming it to

Reviewer: From this line onwards the indentation is incorrect. At least the first line of each paragraph and the first line of each bullet point should be indented four spaces so that it aligns to the right of the item number (`4.`). I recommend to indent all of the lines 4 spaces, to make it clearer in the markdown file that this content is part of listed item, rather than new content after the list.

something descriptive — such as referencing the issue or feature you’re working

> on makes it much easier to find and manage your codespaces, especially if
> you use several.
> 
> To rename your codespace:**

Reviewer: ```suggestion
on - makes it much easier to manage your codespaces:
```

Try to keep the text succinct (so people bother to read it!) and avoid repetition

- Open your [GitHub Codespaces dashboard](https://github.com/codespaces) in
  your browser.
- Find the codespace you just created (it will show the repository name and
  the current random display name).
- Click the ellipsis (**...**) next to your codespace to open the options menu.
- Select **Rename** from the dropdown.

> - Enter a descriptive new display name (e.g., `r-dev-env-issue-229` or
>   `r-dev-env-patch-naming`) and confirm.

Reviewer: ```suggestion
- Enter a descriptive new display name, e.g., `Bug 7084 - text() recycling problems` or `Ignore deprecated/defunct fns in checkS3methods`, and confirm.
```

Remember the container is intended for working on bugs/features in R, not issues with the R dev container. It is not necessary to include `r-dev-env` in the name, as this is already shown (as you point out in the second bullet point here).

> ![rename codespace](../../assets/rdev3_5.png)

Reviewer: As this screenshot is intended to show how to select 'Rename', it would be good to have the current name be three random words. Also it would be great if you could find a tool that lets you take screenshots that show the cursor, as the change in colour under 'Rename' is not obvious. If this is not possible, perhaps you could post-edit the screenshot to add a red oval around the 'Rename' menu item. It would also be good to switch to light mode as the previous screen shot is in light mode (I fear we are not consistent across the docs, but I think it would be good be consistent on the same page). Finally, while I am a fan of screenshots where you can read the text, this is a bit too zoomed in. Aim for a zoom level where the regular text (like 'Rename') is a similar size to the normal text in the docs.

> **Tip:**
> You can rename your codespace at any time. The display name is

>   for your convenience and does not affect your code or environment.

Reviewer: You should use an admonition block of type "tip" here, see https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:tip
